src/main/java/com/mealmentor/enterprise/MealMentorController.java

```diff
@@ -74,8 +78,8 @@ public String read(Model model){
     }
 
 
-    @RequestMapping (value = "/searchReceipe")
-    public String searchReceipe (Recipe Recipe)
+    @RequestMapping (value = "/searchRecipe")
+    public String searchRecipe (Recipe Recipe)
```

Comment:
Just a simple typo here, would have caused a problem when you go to make the page.